client/src/components/CreateQuestion.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+const CreateQuestion = () => {
+  // const []
+  return (
+    <>
+      <form className="w-[70%]  border-gray100 border-4   mt-10 py-6 px-10 rounded-lg mx-auto">
+        <div className="flex flex-col">
+          <h3 className=" text-2xl font-semibold">
+            <span className="text-2xl text-orange">•</span> Create New Question
+          </h3>
+          <hr className="bg-orange h-1" />
+          <textarea
+            name="question"
+            cols="10"
+            rows="5"
+            className="my-4 outline-none border-blue border-2  rounded-lg p-2 text-xl text-darkGray"
+            placeholder="type your question here..."
+          />
+          <textarea
+            name="answer"
+            cols="10"
+            rows="5"
+            className="my-2 outline-none border-blue border-2 rounded-lg p-2 text-xl text-darkGray"
+            placeholder="type your question here..."
```

Comment:
Change bottom box "type your question here..." colour to match sigma design.

Placeholder font colour is fine but the text for user input needs to be much darker for both boxes.